platform/authorized_api_guide.md

```diff
@@ -20,40 +21,317 @@ API credentials which then allows you to request for a token that is used for au
   Name must contains at least 3 characters.
 * Copy your newly generated `CLIENT ID` and `CLIENT SECRET` because they **won't be available for you after dismissing this window.**
 
-### Create access token
+This credentials will be valid as long as you revoke them in your profile.
```

Comment:
I have problem with understanding this sentence:
* Linguistic: "This credential ..." or "Those credentials ..." are correct, "This credentials ..." is not.
* Logic: Credentials will be valid if I'll revoke them or as long as I wont revoke them?